hw/ip/kmac/data/kmac.hjson

```diff
@@ -387,6 +387,9 @@
           desc: "Message FIFO Full indicator"
         }
       ]
+      // At chip level, rom_ctrl runs KMAC on startup so this register won't
+      // have its reset value by the time any CSR tests check it.
+      tags: ["excl:CsrAllTests:CsrExclCheck"]
```

Comment:
I suppose delaying the CSR tests to start until the ROM checker is done is not going to help?

Reply:
@udinator, FYI
This impacts KMAC IP level testbench too.

Reply:
I don't think it would help with this register: the problem is that KMAC might do something and change the register value before the CSR tests start. It will help with all the other registers (as discussed in #6747 - good idea!).

Re the KMAC IP-level testbench: Yes, this exclude will apply there too (of course). Maybe we need some sort of "only exclude at chip level" tag?

Reply:
This is just the status register. I am sure it is verified at the IP level for correctness. I am fine with having this exclusion at the IP level as well. Let's keep it the way it is. 

You have applied the exclusion to the entire CSR. Could you only apply it to the specific fields that are mismatching? At least the test will read the CSR and check the other bits.

Reply:
Ah, good idea. I'll do that.

Reply:
Well, that's weird. Rebasing the PR and everything passes without the exclusion. I ran 10 seeds locally to check it wasn't fluke, and all seems well. This is a little suspicious to me, but I guess we may as well merge and figure out what explodes later.